samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,144 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio as the constant RX or TX carrier, the modulated TX carrier, or the RX or TX sweep.
+
+Overview
+********
+
+The tests are controlled with Command Line Interface (CLI) by the serial port.
+At any time during the tests, you can set the radio parameters output power, bit rate, and channel.
+You can also set the time on each channel in the sweep mode, in steps of 1 millisecond to 99 milliseconds, every 1 millisecond.
+The application also allows you to send a data pattern to another board.
+
+The application starts with enabling the high frequency crystal oscillator and configuring CLI.
+Calling a console command causes the appropriate action.
+
+.. note::
+  For the IEEE 802.15.4 mode, the start channel and the end channel must be within the channel range of 11 to 26.
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
+
+    * Network Core
```

Comment:
For dual core chips, it would be useful if samples documented what should be loaded on each core for the sample to work correctly.

Reply:
Yes, we are working on adding this piece to the doc.

Reply:
Also this will be probably mentioned in the nRF53 User Guide, which needs an update, see NCSDK-4204